contracts/BSCValidatorSet.sol

```diff
@@ -207,151 +228,160 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
       }
     }
 
-    if (index>0) {
-      Validator storage validator = currentValidatorSet[index-1];
+    if (index > 0) {
+      Validator storage validator = currentValidatorSet[index - 1];
       if (validator.jailed) {
-        emit deprecatedDeposit(valAddr,value);
+        emit deprecatedDeposit(valAddr, value);
       } else {
         totalInComing = totalInComing.add(value);
         validator.incoming = validator.incoming.add(value);
-        emit validatorDeposit(valAddr,value);
+        emit validatorDeposit(valAddr, value);
       }
     } else {
       // get incoming from deprecated validator;
-      emit deprecatedDeposit(valAddr,value);
+      emit deprecatedDeposit(valAddr, value);
     }
   }
 
   function jailValidator(Validator memory v) internal returns (uint32) {
     uint256 index = currentValidatorSetMap[v.consensusAddress];
-    if (index==0 || currentValidatorSet[index-1].jailed) {
+    if (index == 0 || currentValidatorSet[index - 1].jailed) {
       emit validatorEmptyJailed(v.consensusAddress);
       return CODE_OK;
     }
     uint n = currentValidatorSet.length;
-    bool shouldKeep = (numOfJailed >= n-1);
+    bool shouldKeep = (numOfJailed >= n - 1);
     // will not jail if it is the last valid validator
     if (shouldKeep) {
       emit validatorEmptyJailed(v.consensusAddress);
       return CODE_OK;
     }
     numOfJailed ++;
-    currentValidatorSet[index-1].jailed = true;
+    currentValidatorSet[index - 1].jailed = true;
     emit validatorJailed(v.consensusAddress);
     return CODE_OK;
   }
 
-  function updateValidatorSet(Validator[] memory validatorSet) internal returns (uint32) {
+  function updateValidatorSet(Validator[] memory validatorSet, ValidatorExtra[] memory _validatorExtraSet) internal returns (uint32) {
```

Comment:
change to `addressp[] memory _validatorBlsKeys`

Reply:
Fixed